src/reducers/dashboard/index.js

```diff
@@ -51,9 +52,15 @@ export function fetchDashboard(dispatch) {
 }
 
 function dashboardRequest(dispatch) {
-  return fetch(`${apiPath}/sessions`)
+  return fetch(`${apiPath}/sessions`, {
```

Comment:
I did not require authentication when getting the sessions by design, but you are free to do so. Sending cookies won't harm anything.